### PR TITLE
For an issue where i always endup being redirected to the same auth.html screen

### PR DESCRIPTION
**Pull request by ShekharReddy4**
Hey @alexisduque look into this and review this :)

### CODE REVIEW

File: auth.html [hunk 1]
```diff
@@ -7,7 +7,7 @@
   <div class="wrapper">
     <form class="form-signin" action="auth.php" method="POST">       
       <h2 class="form-signin-heading">Sign in with your ID</h2>
-      <input type="text" class="form-control" name="user" placeholder="Email Address" required="" autofocus="" /><br>
+      <input type="text" class="form-control" name="user" placeholder="User Name" required="" autofocus="" /><br>
```
Comment (ShekharReddy4) [inline auth.html:10]: This is really what i intend to change :-)
(There are some formatting changes please ignore them )

if the user enters the email id, he will not be able to login. Jus checkout the code in auth.php :)

**Comment by alexisduque** [comment]
Before I merge it, can you squash your commits into a single one, and remove useless diff due to formatting change ? Then, update your PR :smile:

**Comment by ShekharReddy4** [comment]
@alexisduque  i tried to squash them before but git ignore files are aborting squash, Since I don have much time right now, I'll try doing them later :)

Thanks

**Comment by alexisduque** [comment]
Don't worry :smirk:  

thanks :+1:

**Comment by ShekharReddy4** [comment]
as per your comments/requirements, I will create a new PR with the changes proposed. :smile:

**Pull request closed without merging**